### PR TITLE
Clean up admin templates

### PR DESCRIPTION
**Pull request by psren**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | no |
| BC breaks?      | yes (some template events changing) |
| Related tickets | partially #8313 |
| License         | MIT |

### CODE REVIEW

File: src/Sylius/Bundle/ShopBundle/Resources/views/Checkout/address.html.twig [hunk 1]
```diff
@@ -2,40 +2,21 @@
 
 {% form_theme form '@SyliusUi/Form/theme.html.twig' %}
 
-{% block content %}
```
Comment (lchrusciel) [inline src/Sylius/Bundle/ShopBundle/Resources/views/Checkout/address.html.twig]: Changing blocks name should also be mentioned in UPGRADE.md

Comment (psren) [inline src/Sylius/Bundle/ShopBundle/Resources/views/Checkout/address.html.twig]: yeah, will have time at the end of the week. It is planned not to rename the block at all, i'll revert that change

**Comment by psren** [comment]
This one is finished imo.
I will add the info-box to the demo-shop as soon this one is merged, or should i do the pr on the demo, now?

File: src/Sylius/Bundle/ShopBundle/Resources/views/Checkout/address.html.twig (hunk 1 above)
Comment (psren) [inline src/Sylius/Bundle/ShopBundle/Resources/views/Checkout/address.html.twig]: i documented the change now

**Comment by lchrusciel** [comment]
The PR needs rebase. Also, can you squash some of the commits? 14 commits for 7 files is a little bit too much. 

Regarding your question, I would do it in separate PR but first, let's discuss if we want to put this change in the core. At the end changing blocks can slightly break existing websites.

**Comment by pamil** [comment]
Thanks for the effort put into this PR! I'm sorry we've missed it on the way to stable release, we aren't able to merge it now because it contains BC breaks.

**Pull request closed without merging**